README.md

```diff
@@ -106,6 +106,34 @@ item.created? # => true
 item.selected? # => false
 ```
 
+### Filter methods
+
+Plugin can define dataset methods for all enum values:
+
+```ruby
+Item.plugin :enum_values, filter_methods: true # default is `false`
```

Comment:
A small naming suggestion, we can discuss it, but this is what I saw:

You're calling them `dataset methods`, it's actually `Dataset`'s methods, but you named the option `filter_methods`. A bit confusing, and I see no problem with `dataset_methods` option: I don't think that there will be other dataset methods, and it's all about enums.

What do you think?

Reply:
i can do it, but for me it's like change name of `predicate_methods` to `instance_methods`. We can't filter an instance, only collection or dataset, so that's why `filter_methods` was used

Reply:
i thought that `dataset_methods` isn't clear enough naming.

Reply:
My response (with agreement and suggestion) somwhy moved below: https://github.com/AlexWayfer/sequel-enum_values/pull/26#discussion_r606235753